hub/columns/miscellaneous/mxp.tex

```diff
@@ -17,9 +17,6 @@
 		``the active number of words in memory (counting continuously from position $0$)'' \ob{TODO: reference to \cite{EYP-London}, 9.4.1. Machine State};
 	\item \miscMxpGasMxp{}:
 		memory expansion cost + whatever ``linear'' cost the instruction may incur;
-	\item \miscMxpTypeFourInstMayTriggerMmu{}
-		binary column which may only light up for \mxpMod{} instruction of ``\textbf{type 4}'';
-		see section~(\ref{mxp: may trigger non trivial operation});
```

Comment:
`\miscMxpTypeFourInstMayTriggerMmu` is completetely subsumed by `\miscMxpSizeOneNonzeroNoMxpx`.